include/mbgl/sprite/sprite_image.hpp

```diff
@@ -22,8 +22,8 @@ class SpriteImage : private util::noncopyable {
     // Whether this image should be interpreted as a signed distance field icon.
     const bool sdf;
 
-    float getWidth() const { return image.width / pixelRatio; }
-    float getHeight() const { return image.height / pixelRatio; }
+    float getWidth() const { return image.size.width / pixelRatio; }
```

Comment:
In this particular case, we're using `float`s to represent the width/height, but our `Size` type uses integers, so we should keep this.